### PR TITLE
467 player and outfit counts look wonky on mobile

### PR DESCRIPTION
**Pull request by oslohes123**
Updated on styling need review on how to change it further

Or maybe a simpler version may be 
image
This is on Iphone SE which has a very small x-axis
image
Looks much nicer on a larger phone where the whole line is visible
And on ipad
image
And then on PC it looks fine
image
Also this reduces the code complexity a bit since no breakpoints are needed as these are the same styles that are themselves adapting to the size of the display

### CODE REVIEW

File: components/alert/AlertCharacterMetrics.vue [hunk 1]
```diff
@@ -10,8 +10,9 @@
       <h1>Loading...</h1>
     </div>
     <div v-if="loaded" class="grid grid-cols-12">
-      <div class="col-span-12 mb-2 flex justify-center">
-        <div class="pr-2 py-2">Player Counts:</div>
+    
+      <div class="col-start-1 col-span-full mb-2 flex flex-wrap justify-center ">
+        <div class = "p-2">Player Counts</div>
```
Comment (Maelstromeous) [inline components/alert/AlertCharacterMetrics.vue:15]: Extra spacing here for no reason

File: components/alert/AlertCharacterMetrics.vue [hunk 2]
```diff
@@ -10,8 +10,9 @@
       <h1>Loading...</h1>
     </div>
     <div v-if="loaded" class="grid grid-cols-12">
-      <div class="col-span-12 mb-2 flex justify-center">
-        <div class="pr-2 py-2">Player Counts:</div>
+    
+      <div class="col-start-1 col-span-full mb-2 flex flex-wrap justify-center ">
```
Comment (Maelstromeous) [inline components/alert/AlertCharacterMetrics.vue:14]: Not sure if `col-start-1` is required if you're also using `col-span-full`, think it's redundant. ALso note extra space on the end

**Review by Maelstromeous — CHANGES_REQUESTED** [review]
Looking much better yeah, just a few minor details to fix

**Comment by oslohes123** [comment]
There shouldn't be any extra spaces I couldn't find the second one you said, if you view the file itself in the branch there are no unecessary spaces that I can see. Also I have removed the col-start-1 as col-span-1 translates to 1 / -1 where 1 tells the div to start at the beginning anyway cheers !

**Comment by Maelstromeous** [comment]
There's still extra spaces on line 14

**Comment by oslohes123** [comment]
> There's still extra spaces on line 14

How about now?

**Review by Maelstromeous — APPROVED** [review]
LGTM

**Pull request merged**